tasks/build_tags.py

```diff
@@ -124,7 +124,7 @@
 SERVERLESS_TAGS = {"serverless", "otlp"}
 
 # SYSTEM_PROBE_TAGS lists the tags necessary to build system-probe
-SYSTEM_PROBE_TAGS = AGENT_TAGS.union({"clusterchecks", "linux_bpf", "npm"}).difference({"python", "systemd"})
```

Comment:
Are you sure about removing this ? For example a lot of the CO-RE work in the system probe is gated on this

Reply:
I reverted this change.